### DISCUSSION TITLE
Send bitmap to display

### DISCUSSION THREAD

**Discussion by wbeck77** [arendst/Tasmota#15645] [Q&A] [answered]
Hi all,

I would like to show a bitmap on a connected SSD1306 display. 
In best case I can send the bitmap using MQTT to my Tasmota device.
It would be perfect to "embed" a text/value afterwards. 

I´m wondering if such a feature is already supported. 
At least I can't find any documentation about displaying bitmaps.....

Best regards

Wolfgang

**Comment by s-hadinger** [comment]
Are you using ESP32? If so LVGL might be an answer

**Comment by wbeck77** [comment, marked as answer]
I’m using a Sonoff S20 with ESP8266 :-/

**Reply by Jason2866** [reply]
Possible with https://tasmota.github.io/docs/Scripting-Language/ Some learning ahead
